### PR TITLE
Update revisions of RadeonOpenCompute and Khronos ICD

### PR DESCRIPTION
**Pull request by wkwchau**
Wilkin wants to merge 1 commit into RadeonOpenCompute:roc-2.3.x from RadeonOpenCompute:roc-2.3.x-man-update.

### CODE REVIEW

File: opencl.xml [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
 
-  <default remote="RadeonOpenCompute" revision="master" sync-c="true" sync-j="4"/>
+  <default remote="RadeonOpenCompute" revision="refs/tags/roc-2.3.0" sync-c="true" sync-j="4"/>
 
   <project path="opencl" name="ROCm-OpenCL-Runtime"/>
   <project path="opencl/compiler/driver" name="ROCm-OpenCL-Driver"/>
-  <project path="opencl/compiler/llvm" name="llvm" revision="amd-common"/>
-  <project path="opencl/compiler/llvm/tools/clang" name="clang" revision="amd-common"/>
-  <project path="opencl/compiler/llvm/tools/lld" name="lld" revision="amd-common"/>
+  <project path="opencl/compiler/llvm" name="llvm"/>
```
Comment (kzhuravl) [inline opencl.xml]: revision for llvm must be
  - revision=""refs/tags/roc-ocl-2.3.0"

File: opencl.xml [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
+  <project path="opencl/compiler/llvm/tools/clang" name="clang"/>
+  <project path="opencl/compiler/llvm/tools/lld" name="lld"/>
```
Comment (kzhuravl) [inline opencl.xml]: revision for lld must be
  - revision=""refs/tags/roc-ocl-2.3.0"

Comment (wkwchau) [reply opencl.xml]: Should the revision for clang be provided as well or only for llvm and lld?

Comment (kzhuravl) [reply opencl.xml]: only llvm and lld are refs/tags/roc-ocl-2.3.0, others (including clang) are as usual. So, to answer your question: only llvm and lld. Thanks.

**Pull request merged**